stories/ControlledChipInput.js

```diff
@@ -31,6 +31,11 @@ class ControlledChipInput extends React.Component {
       value={this.state.chips}
       onRequestAdd={(chip) => this.handleRequestAdd(chip)}
       onRequestDelete={(deletedChip) => this.handleRequestDelete(deletedChip)}
+      onBlur={(event) => {
+        if (this.props.addOnBlur && event.target.value) {
```

Comment:
Somehow didn't notice the props were alphabetically ordered! :blush: All done now